packages/bp/src/admin/ui/src/workspace/bots/CreateBotModal.tsx

```diff
@@ -119,7 +119,7 @@ class CreateBotModal extends Component<Props, State> {
     }
 
     try {
-      await api.getSecured().post('/admin/workspace/bots', newBot)
+      await api.getSecured({ timeout: 20000 }).post('/studio/manage/bots/create', newBot, { baseURL: window.ROOT_PATH })
```

Comment:
Why add a 20sec timeout only now?

Reply:
Bot creation is longer since we added the lock file, sometimes it times out. Could be even bigger I guess